python/lsst/meas/deblender/plugins.py

```diff
@@ -296,12 +298,10 @@ def buildMultibandTemplates(debResult, log, useWeights=False, usePsf=False,
             continue
 
         # Temporary for initial testing: combine multiple components
-        model = blend.get_model(m=pk, flat=False)
-        model = model.astype(np.float32)
+        model = blend.get_model(k=pk).astype(np.float32)
 
         # The peak in each band will have the same SpanSet
-        mask = afwImage.Mask(np.array(np.sum(model, axis=0)>0, dtype=np.int32),
-                             xy0=debResult.footprint.getBBox().getBegin())
+        mask = afwImage.Mask(np.array(np.sum(model, axis=0)>0, dtype=np.int32), xy0=xy0)
```

Comment:
Since you're changing this line anyway, it's good practice to make it comply with our coding standards even if the original infraction wasn't yours.